benchmarks/ops/benchmark_ttt.py

```
@@ -2,12 +2,13 @@
 # pip install "git+https://github.com/openai/triton.git#egg=triton&subdirectory=python"
 
 import torch
-from benchmark import benchmark_combined, benchmark_forward, benchmark_backward
+from benchmark import benchmark_backward, benchmark_combined, benchmark_forward
 
-from fla.ops.gla import chunk_gla
 from fla.ops.delta_rule import chunk_delta_rule
+from fla.ops.gla import chunk_gla
 from fla.ops.ttt import chunk_ttt_linear, fused_chunk_ttt_linear
 from fla.ops.ttt.naive import chunk_ttt_linear_ref
```

Ruff (0.8.2)

`fla.ops.ttt.naive.chunk_ttt_linear_ref` imported but unused

Remove unused import: `fla.ops.ttt.naive.chunk_ttt_linear_ref`

(F401)